docs/pi-scripts/jackrun.bash

```diff
@@ -25,12 +25,13 @@ else
   cmp -s $WEBVER $LOCALVER
   if [ "$?" -ne "0" ]; then
     # There is a new version on the web  Stash the old one
-    echo "New version available"
+    echo "New version available, gonna download"
     mkdir -p rollback
```

Comment:
It might be simpler to skip the new dir and just move the file to **$PROGRAM.rollback**. The rollback dir makes more sense if there are more than one files involved.

Reply:
Ideas was to keep old versions in rollback at some point but why design that when we its not required.